src/interface/src/app/treatments/direct-impacts/metric-filters/metric-filters.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="grid-container">
```

Comment:
you can save one node and avoid this `div`, and just move the styles of `grid-container` to `:host`

Reply:
updated